__tests__/itinerary.test.js

```diff
@@ -0,0 +1,13 @@
+const Itinerary = require("../src/itinerary");
+
+describe("Itinerary", () => {
+  it("returns an object", () => {
+    expect(new Itinerary()).toBeInstanceOf(Object);
+  });
+  it("sets the ports property of the itinerary", () => {
```

Comment:
Its common convention to leave a clear line between each of your tests, just to make it easier to read at a glance
```
  });

  it("sets the ports property of the itinerary", () => {
```